content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,195 @@ description: >
+
+In all scenarios involving inter-node Pod networking (physical or virtual), you will have a physical network in place. This provides you with communications between different network resources needed for K8s and non-K8s application support, APIs, metrics collection, operations and troubleshooting. 
+
+For a K8s-specific example, you could have a process call the control plane API server to issue an HTTP request to a node kubelet. 
+
+Figure 4 depicts the physical network supporting this API example.
+
+{{< figure src="/docs/images/k8s-api-to-kubelet-example.drawio.svg" alt="diagram of k8s API server to node kubelet" class="diagram-large" caption="Figure 4. Example of control plane to node communications using the physical network" >}}
+
+In addition to addressing your K8s inter-node Pod networking requirements, your physical network can support NTP, IPsec IKEv2, OAM tasks and others specific to your environment.
+
+{{< note >}}
+The physical network can transport encapsulated (used in virtual overlay networks) and non-encapsulated pod packets.
+{{< /note >}}
+
+#### Pods on different nodes using a physical network
+
+This example uses Pod and root network namespaces, veth links, network plugins and the physical network. 
+
+Figure 5 illustrates an example where Pods on different nodes use the physical network to communicate.
+
+{{< figure src="/docs/images/k8s-net-phys-net.drawio.svg" alt="diagram of inter-node Pod networking using physical network " class="diagram-large" caption="Figure 5. Example of Pod 1 - Pod 2 networking on different nodes using a physical network" >}}
+
+The network plugin exposes Pods (pod IP) to the physical network addressing and forwarding. In essence your Pods become IP host end-points reachable by all other end-points on the physical network. You can leverage existing network transport, address management, performance, security and operational best-practices to support your Pod networking requirements.
+
+To ensure Pod access integrity, you should apply [pod security](/docs/concepts/security/pod-security-standards) measures along with one or more network functions designed to protect hosts from malicious attacks. Examples include packet filters, address management, network segmentation, authentication and encryption. 
+
+The physical network approach offers you several advantages:
+
+* Common physical network to handle K8s and non-K8s traffic.
+* Optimal network performance.
+* Enhanced security with a CNI plugin that handles encrypted communications such as IPsec or TLS.
+
+The challenges to you include CNI plugin management and non-K8s network operations impact to your cluster.
+
+For more information on two examples of physical network implementations, see [Calico BGP](https://github.com/projectcalico/calico) and [Cilium native-routing](https://docs.cilium.io/en/v1.13/network/concepts/routing/#native-routing).
```

Comment:
There's a detail missing here: the bridging. The node is using a software bridge to make each Pod appear as if it has a physical connection. The node still only has one network interface.

Reply:
Is the software bridge the virtual L2 bridge or something different?

Reply:
The text is what could mislead; eg: _The network plugin exposes Pods (pod IP) to the physical network addressing and forwarding._

That's where I'd mention the use of a bridge. There actually are designs where each Pod gets its own network interface that can talk directly to things outside the node. This isn't one of those designs.